src/app/models/AudioRecording.ts

```diff
@@ -100,15 +104,28 @@ export class AudioRecording
     return listenRecordingMenuItem.route.format({ audioRecordingId: this.id });
   }
 
+  public constructor(data: IAudioRecording, injector?: Injector) {
+    super(data, injector);
+
+    // Set timezone on recorded date
+    if (this.recordedDateTimezone) {
+      this.recordedDate = this.recordedDate.setZone(this.recordedDateTimezone);
```

Comment:
add a note to revert this.

If the recording were to be saved back to the database, there is a chance it could be saved in the wrong format since it is not longer a UTC date. How is it serialized? 

Even if the above point is not valid (it should not be), all dates are in UTC in the database, in the server. We should avoid changing the semantic meaning of this field.

Reply:
> all dates are in UTC in the database, in the server

This sounds a lot more like a server issue than a front end issue. If dates should be stored as UTC in the database, this should be handled server-side. Doubly so if the reason you say this is because some values on the server are stored with a timezone instead of having the timezone stored separately (I'm not sure if any such fields exist). Controlling this logic client side sounds like a problem waiting to happen, especially as we expose and expect users to potentially directly use the API.

Reply:
Time zones are not offsets.  Time zones cannot be stored in date stamps.

If you send us date with an offset we will convert the date back to UTC when it is stored.

Regardless, there is no point modifying this field if we have a plan for showing these dates consistently, which for whatever reason you've chosen not to employ here 

Mark this as revertible, log a note in #1736 and move on.

We can discuss further later